eng/scripts/automation_init.sh

```diff
@@ -57,4 +57,7 @@ cat > $outputFile << EOF
     "GOROOT": "$GOROOT"
   }
 }
-EOF
\ No newline at end of file
+EOF
+
```

Comment:
Can you remove line#14 'set -x' which would print all the commands and arguments to the stderr stream? This will print all the commands and logs in the pipeline logs including a lot of module download status, which is not useful. We can easily know what command being run by looking at the shell script for trouble-shooting purpose.